src/os/bluestore/BlueStore.cc

```diff
@@ -6298,7 +6447,7 @@ void BlueStore::_dump_alloc_on_failure()
     cct->_conf->bluestore_bluefs_alloc_failure_dump_interval;
   if (dump_interval > 0 &&
     next_dump_on_bluefs_alloc_failure <= ceph_clock_now()) {
-    shared_alloc.a->dump();
+    alloc->dump();
     next_dump_on_bluefs_alloc_failure = ceph_clock_now();
```

Comment:
`_dump_alloc_on_failure()` is dead code now, but I think it should refer `shared_alloc.a`, not BlueStore Block (private) data.